client/src/app/components/CreateProject/Options/Option.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import "./options.css";
+const classNames = require("classnames");
+
+class Option extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isActive: this.props.isActive };
+  }
+
+  componentDidMount() {
+    if (this.props.isDefault === this.props.name) {
+      this.setState({ isActive: true });
+      this.props.checkMax(1, this.props.name);
+    }
+  }
+
+  componentWillUpdate(props) {
```

Comment:
It doesn't matter so much in this project, but `componentWillUpdate` is now considered unsafe in the react documentation. You can generally achieve the same thing with `componentDidUpdate`

Reply:
I will leave this as is but take your comment into account for the future.